TestSuites/ProjectSuit.robot

```diff
@@ -2,16 +2,15 @@
 Library           Selenium2Library
 Resource          ../service/UI/ProjectService.robot
 Resource          ../globalConfig/Common.robot
-Library           ../utils/TestDataGen.py
-Library           ../page/ProjectPage.py
 Resource          ../service/UI/LoginService.robot
+Resource          ../utils/TestDataGenerator.robot
 
 *** Test Cases ***
 CreateProject
     [Documentation]    The test is intended to check the creation of a project via Project Page.
     [Setup]    Run Keywords    Set Random String
     ...    AND    Login And Check Top Logo Visible UI    ${ADMIN_LOGIN}    ${ADMIN_PASSWORD}
     ...    AND    Open Project Page UI
-    Create Project UI    ${rndmString}
-    Check Created Project UI    ${rndmString}
+    Create Project UI    ${randomString}
+    Check Created Project UI    ${randomString}
```

Comment:
A service keyword may check is project exist or not. Check is an option of a tests (validation). This new Is Project Exist UI service's keyword will return boolean true or false (prefix 'Is' means that boolean value will be returned)